addons/istio/1.5.x/istio-1.yaml

```diff
@@ -6,25 +6,24 @@ metadata:
   labels:
     kubeaddons.mesosphere.io/name: istio
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.4.3-2"
-    appversion.kubeaddons.mesosphere.io/istio: "1.4.3"
-    appversion.kubeaddons.mesosphere.io/kiali: "1.4.3"
-    appversion.kubeaddons.mesosphere.io/jaeger: "1.4.3"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.5.4-1"
+    appversion.kubeaddons.mesosphere.io/istio: "1.5.4"
+    appversion.kubeaddons.mesosphere.io/kiali: "1.15.1"
+    appversion.kubeaddons.mesosphere.io/jaeger: "1.16.0"
     stage.kubeaddons.mesosphere.io/kiali: Preview
     stage.kubeaddons.mesosphere.io/jaeger: Preview
     endpoint.kubeaddons.mesosphere.io/kiali: "/ops/portal/kiali"
     endpoint.kubeaddons.mesosphere.io/jaeger: "/ops/portal/jaeger"
     docs.kubeaddons.mesosphere.io/istio: "https://istio.io/docs/"
     docs.kubeaddons.mesosphere.io/kiali: "https://istio.io/docs/tasks/telemetry/kiali/"
     docs.kubeaddons.mesosphere.io/jaeger: "https://istio.io/docs/tasks/telemetry/distributed-tracing/jaeger/"
-    values.chart.helm.kubeaddons.mesosphere.io/istio: "https://raw.githubusercontent.com/mesosphere/charts/63bbc17eda76f2136f6ab4f6d6eef7e764e5f0a5/staging/istio/values.yaml"
 spec:
   namespace: istio-system
   requires:
     - matchLabels:
         kubeaddons.mesosphere.io/name: cert-manager
   kubernetes:
-    minSupportedVersion: v1.15.6
+    minSupportedVersion: v1.16.0
```

Comment:
Do we expect any issues for customers still running K8s 1.15 and upgrading Konvoy? I'm not to experienced with our supported upgrade scenarios re Addons.

Reply:
Istio is still in a preview with no upgrade support